.github/workflows/helm-weekly-release-reviewer.yml

```diff
@@ -48,13 +32,12 @@ jobs:
           --approve -b "**I'm approving** this pull request, since it is a helm release."
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
-          GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
i thought we decomissioned using GITHUB_TOKEN PAT. Or is this the ephemeral token that gets issued for every action run?